src/itemrenderer.cpp

```diff
@@ -130,17 +142,40 @@ void render_html(
 	}
 }
 
+void item_renderer::render_plaintext(
+	const std::string& source,
+	std::vector<std::pair<LineType, std::string>>& lines)
+{
+	std::string::size_type pos = 0;
+	while (pos < source.size()) {
+		// Internally, we only use '\n' as the newline character
+		const auto end_of_line = source.find_first_of("\n", pos);
+		const std::string line = source.substr(pos, end_of_line - pos);
+		lines.push_back(std::make_pair(LineType::wrappable, line));
+		if (end_of_line == std::string::npos) {
```

Comment:
If `end_of_line` is `npos` won't this break the `substr` call above? Should this check not be above the substr call?

Reply:
It won't break `substr` because `substr` only returns as much data as the source string has.